xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -2124,8 +2180,9 @@ void CMusicInfoScanner::SetDiscSetArtwork(CAlbum& album, const std::vector<std::
   discnumber in each subfolder - save the local art (all types) for each disc.
   */
 
-  if (paths.size() <= 1)
-    return;  // No disc subfolders to process
+//  if (paths.size() <= 1)
+//    return;  // No disc subfolders to process
+//  Keep scanning - sets can be in one folder using extraart
```

Comment:
Humm...... use of extraart approach (rather than per folder) is something to be considered generically for more than just box sets. I need to check deeper.